coss/club/models.py

```diff
@@ -36,6 +36,24 @@ class Testimonial(models.Model):
     quote = RichTextField('Testimonial quote')
 
 
+class ClubsPage(Page, RichText):
```

Comment:
Since this is a copy paste from the `homepage` we don't need another model. We can just a different page from the admin. I would suggest to rename the homepage to something like `BaseClubPage` or similar.